### DISCUSSION TITLE
研究生模板信息栏下方添加“摘要”标题

### DISCUSSION THREAD

**Discussion by SJHNJU** [nju-lug/NJUThesis#99] [General]
在中英摘要章节，摘要正文上方没有“摘要”和“Abstract”标题，应当如何添加呢

**Comment by atxy-blip** [comment]
是因为现在模板在 `\mainmatter` 前面都是 plain 样式……………………

这周末发新版，会紧急修一下（

**Reply by SJHNJU** [reply]
感谢！

**Reply by atxy-blip** [reply]
现在 v0.16.0 的页眉应该都是正确的。

**Reply by SJHNJU** [reply]
感谢！

**Comment by atxy-blip** [comment]
我看错了，以为是页眉问题。<https://github.com/nju-lug/NJUThesis/commit/0f2cf7c8ae4e4bd60e43a0cb645d82182b01b637> 添加了你需要的小标题，马上会随着 0.16.1 这个小版本发布。

**Reply by SJHNJU** [reply]
嗯嗯没关系的，谢谢了

**Comment by zhoujian9410** [comment]
在新版出来前，我是这么解决的。
摘要正文前分别加
```latex
%中文摘要正文前
\begin{center}
	\zihao{-3}
	\textsf{摘 \quad 要}
\end{center}
```
和
```latex
%英文摘要正文前
\begin{center}
	\zihao{-3}
	\textsf{\uppercase{abstract}}
\end{center}
```

**Reply by SJHNJU** [reply]
OK了，非常感谢！
